aten/src/ATen/CMakeLists.txt

```diff
@@ -373,7 +373,7 @@ TARGET_LINK_LIBRARIES(ATen cpuinfo)
 # ---[ Configure SLEEF
 IF(NOT TARGET sleef)
   add_subdirectory("cpu/sleef")
-  include_directories(${CMAKE_BINARY_DIR}/include)
+  include_directories(SYSTEM ${CMAKE_BINARY_DIR}/include)
```

Comment:
Nit: use TARGET_INCLUDE_DIRECTORIES to try to best follow modern CMake guidelines.

Reply:
Yeah ATen CMake is very bad with these things. We are rewriting our CMake stuff into one unified build anyway, so hopefully all these things will be fixed to better align with best practices